Addons/Entitas.CodeGeneration.Unity.Editor/Entitas.CodeGeneration.Unity.Editor/Editor/UnityCodeGenerator.cs

```diff
@@ -63,7 +63,7 @@ public static class UnityCodeGenerator {
             }
 
             var assembly = typeof(UnityEditor.Editor).Assembly;
-            var logEntries = assembly.GetType("UnityEditorInternal.LogEntries");
+            var logEntries = assembly.GetType("UnityEditor.LogEntries");
```

Comment:
Included preprocessor condition to ensure targeting of only versions v2017.1.0+.